docs/how-to/networks.md

```diff
@@ -102,16 +102,16 @@ Nerpa is a long-lived devnet, **best for developers building storage apps**. Ner
 | --- | --- |
 |Sector Sizes | 512MiB |
 |Branch | `ntwk-nerpa` |
-|Tag | [`tag ntwk-nerpa-7.7.0`](https://github.com/filecoin-project/lotus/tree/ntwk-nerpa-7.7.0) |
-|Details page | [nerpa.fildev.network](http://www.nerpa.fildev.network/) |
-|Next expected reset | August 12, 2020 |
+|Tag | n/a |
+|Details page | n/a |
```

Comment:
Once Travis' [network-info PR 7 is merged](https://github.com/filecoin-project/network-info/pull/7/files) we can point it to https://network.filecoin.io/ ?

Reply:
The PR is merged but nerpa info doesn't appear to show up on the website. I only see the details for calibration.